mycpp/gc_dict.h

```diff
@@ -43,18 +40,12 @@ class GlobalDict {
  public:
   int len_;
   int capacity_;
-  GlobalSlab<int, N>* entry_;  // TODO: should be sized differently
+  GlobalSlab<int, N>* index_;  // TODO: should be sized differently
```

Comment:
can remove this TODO

Reply:
Actually it's not sized differently now!  So we still have the TODO

I mentioned that in another comment

In theory I think if it's sized the same as keys_ and values_, then the performance will be worse ...